js/styles.js

```diff
@@ -73,13 +73,13 @@ function resetState() {
 
 // on mouseover
 function highlightCounty(selection) {
-  d3.select(selection)
+  selection
     .classed("highlighted-county", true);
 }
 
 // on mouseout
-function unhighlightCounty(selection) {
-  d3.select(selection)
+function unhighlightCounty() {
+  d3.selectAll('.county')
```

Comment:
This looks more computationally intensive than selecting and editing a single element. Is it possible to keep the selection in there, or maybe to call `d3.select('.highlighted-county')` here instead of selecting all the counties?

Reply:
I couldn't figure out how to pass the previous county here (https://github.com/USGS-VIZLAB/water-use-15/pull/172/files#diff-5fd12accec11a9db4351f064170a4fd2R87) without setting some type of global variable, but yes - selecting just that one should work!